diff --git a/packages/zweb-design b/packages/zweb-design
--- a/packages/zweb-design
+++ b/packages/zweb-design
@@ -1 +1 @@
-Subproject commit 8d71fa41165281d124c8f9316ec440528aa3cedd
+Subproject commit 8d71fa41165281d124c8f9316ec440528aa3cedd-dirty
diff --git a/packages/zweb-public-component b/packages/zweb-public-component
--- a/packages/zweb-public-component
+++ b/packages/zweb-public-component
@@ -1 +1 @@
-Subproject commit 07d9b94e1e27ac47206249f20c4453f470972ca5
+Subproject commit 07d9b94e1e27ac47206249f20c4453f470972ca5-dirty
